## 82% of end-end tests have passed
```
Total executed tests: 51
Failed tests: 5
Expected to fail tests: 4
Passed tests: 42
```
```
IDs of failed tests: 727230,702732,702894,703503,727229 
```
```
IDs of expected to fail tests: 703495,702948,727232,703629 
```
<h3>Failed tests (5)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=disable-send-text-input&PR_ID=20564&APK_NAME=20564.apk&TR_CASE_IDS=727230,702732,702894,703503,727229">Rerun failed tests</a></li><br/><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590645">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td><p><blockquote><div># STEP: Getting ETH amount in the wallet of the sender before transaction</div><div>Device 1: Find `WalletTab` by `accessibility id`: `wallet-stack-tab` </div></blockquote></p>

```
critical/test_wallet.py:152: in test_wallet_send_asset_from_drawer
    sender_balance, receiver_balance, eth_amount_sender, eth_amount_receiver = self._get_balances_before_tx()
critical/test_wallet.py:40: in _get_balances_before_tx
    self.wallet_1.wallet_tab.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: WalletTab by accessibility id: `wallet-stack-tab` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590644">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td><p><blockquote><div>Device 1: Swiping right on element SlideButton </div><div>Device 1: Find `SlideButton` by `xpath`: `//*[@resource-id='slide-button-track']` </div></blockquote></p>

```
critical/test_wallet.py:132: in test_wallet_send_eth
    self.wallet_1.send_asset(address=self.receiver['wallet_address'], asset_name='Ether', amount=amount_to_send)
../views/wallet_view.py:130: in send_asset
    self.confirm_transaction()
../views/wallet_view.py:118: in confirm_transaction
    self.slide_and_confirm_with_password()
../views/wallet_view.py:112: in slide_and_confirm_with_password
    self.slide_button_track.slide()
../views/base_view.py:257: in slide
    self.swipe_right_on_element(width_percentage=1.3, start_x=100)
../views/base_element.py:308: in swipe_right_on_element
    location, size = self.get_element_coordinates()
../views/base_element.py:294: in get_element_coordinates
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: SlideButton by xpath: `//*[@resource-id='slide-button-track']` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590599">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><p><blockquote><div>Device 1: `Text` is `You` </div><div>Device 2: `Text` is `user admin` </div></blockquote></p>

```
critical/chats/test_group_chat.py:437: in test_group_chat_pin_messages
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Pinned messages count 4 doesn't match expected 3 for user member_1
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bb6dc50eb5104160b6c31daf1c90c59d?auth=149859083ec55466e1c88fa336284c01#362">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/237543617bb645c09c5626a465ea9102?auth=1d9ec893f4db87f2a651ba9425af4413#230">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5dba910ce02146b683f0bc665c7e24dc?auth=6a35fa81e247524d8b9cf20e3332cb18#229">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590640">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><p><blockquote><div># STEP: Check Status community tags in the Discover communities screen</div><div># STEP: Check Status Inu community tags in the Discover communities screen</div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:300: in test_community_discovery
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community ' 0xUX' is not in the Discover Communities list.
E    Community 'Status' is not in the Discover Communities list.
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/76025d146f884680bf551db06179826f?auth=eb4fffcc77bedfc6fab948729889e8b9#354">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590626">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//*[starts-with(@text,'Hurray! unblocked')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']//*[@content-desc='message-status']/android.widget.TextView` </div><div>Device 2: `Text` is `Sent` </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:767: in test_community_contact_block_unblock_offline
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message was not delivered after back up online.
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/39acbc16ff4848ca8f172e2104d0c2bf?auth=b137c38f17c3007e7a4651c6ccef21be#419">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/429b55b3804741b184530b63f843e2f0?auth=fad00546334058d651220dc8196f6e83#393">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Expected to fail tests (4)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590628">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'#cats')]` </div><div>Device 2: Looking for a message by text: it is just a message text </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1124: in test_community_hashtag_links_to_community_channels
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message in community channel is not visible for user before join 
```

<b>[[Can't enter channel after community is fetched for the first time, https://github.com/status-im/status-mobile/issues/20395]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6791b41168d646beb3bb44bc016eaba1?auth=3114fb5a9cfab212b03201611f27da29#303">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0ef0623a0ba84a9597adc143017cf9a4?auth=8a7ead937f4e7e3e530e8fde34395f0f#292">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590641">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: Button </div><div>Device 2: Looking for community: 'open community' </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1212: in test_community_join_when_node_owner_offline
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message in community channel is not visible for user before join, it was indicated as not sent for the sender before he went offline
E    Joined status is not displayed
E    open community is not listed inside Joined communities tab 
```

<b>[[Can't enter channel after community is fetched for the first time, https://github.com/status-im/status-mobile/issues/20395]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6791b41168d646beb3bb44bc016eaba1?auth=3114fb5a9cfab212b03201611f27da29#502">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0ef0623a0ba84a9597adc143017cf9a4?auth=8a7ead937f4e7e3e530e8fde34395f0f#476">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590638">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><p><blockquote><div># STEP: Change device time so chat will be unmuted by timer</div><div>Device 2: Long press on `ChatElement` </div></blockquote></p>

```
critical/chats/test_group_chat.py:497: in test_group_chat_mute_chat
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Chat is still muted after timeout 
```

<b>[[Chat is not unmuted after expected time: https://github.com/status-im/status-mobile/issues/19627]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bb6dc50eb5104160b6c31daf1c90c59d?auth=149859083ec55466e1c88fa336284c01#482">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/237543617bb645c09c5626a465ea9102?auth=1d9ec893f4db87f2a651ba9425af4413#273">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5dba910ce02146b683f0bc665c7e24dc?auth=6a35fa81e247524d8b9cf20e3332cb18#243">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590647">test_wallet_add_remove_watch_only_account</a>, id: 727232 </b></td></tr><tr><td><p><blockquote><div>Device 1: `Text` is `0x8d2413447ff297d30bdc475f6d5cb00254685aae` </div><div>Device 1: Click system back button </div></blockquote></p>

```
critical/test_wallet.py:256: in test_wallet_add_remove_watch_only_account
    self.home_view.driver.fail(
base_test_case.py:178: in fail
    pytest.fail('Device %s: %s' % (self.number, text))
 Device 1: Incorrect address '0x8d2413447ff297d30bdc475f6d5cb00254685aae' is shown when swiping between accounts, expected one is '0x8d2413447ff297d30bdc475f6d5cb00254685aae' 
```

<b>[[Missing networks in account address, https://github.com/status-im/status-mobile/issues/20166]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/472490f1fa8a47debb10361baebdf320?auth=d0f272b61b544a2f65ce8d2c0e8f08c8#102">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (42)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590604">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/371ef8d1ff514b7db7204a0038aee689?auth=2749684877adf2f4d21a6fb917bb2d52#134">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ff637d414e1942c58113d9b8ba2e4bfe?auth=be44fbefb8452df7f7029f43f0fa06ee#214">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590622">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/371ef8d1ff514b7db7204a0038aee689?auth=2749684877adf2f4d21a6fb917bb2d52#99">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ff637d414e1942c58113d9b8ba2e4bfe?auth=be44fbefb8452df7f7029f43f0fa06ee#121">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590621">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/371ef8d1ff514b7db7204a0038aee689?auth=2749684877adf2f4d21a6fb917bb2d52#56">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ff637d414e1942c58113d9b8ba2e4bfe?auth=be44fbefb8452df7f7029f43f0fa06ee#55">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590627">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cce84edc9c6d4d5f8be544405f679de1?auth=335fbfb46a78429cf93b57f636619315#128">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/86c45793925941b3a7d1dae1bf591247?auth=c6e2e50e354e611380f711eaa7c207d3#98">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590607">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/334c5ffc959d40afab743e0a5d684d66?auth=08dc9511b2313015e5b46a4321e34536#173">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/78913a8187b0468182c7ebd009e4e8b5?auth=bffe3aa760bbf8264e1d0f62c0b2e891#129">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590606">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/334c5ffc959d40afab743e0a5d684d66?auth=08dc9511b2313015e5b46a4321e34536#99">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/78913a8187b0468182c7ebd009e4e8b5?auth=bffe3aa760bbf8264e1d0f62c0b2e891#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590639">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/334c5ffc959d40afab743e0a5d684d66?auth=08dc9511b2313015e5b46a4321e34536#109">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/78913a8187b0468182c7ebd009e4e8b5?auth=bffe3aa760bbf8264e1d0f62c0b2e891#116">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590635">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bb6dc50eb5104160b6c31daf1c90c59d?auth=149859083ec55466e1c88fa336284c01#288">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/237543617bb645c09c5626a465ea9102?auth=1d9ec893f4db87f2a651ba9425af4413#168">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5dba910ce02146b683f0bc665c7e24dc?auth=6a35fa81e247524d8b9cf20e3332cb18#142">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590634">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bb6dc50eb5104160b6c31daf1c90c59d?auth=149859083ec55466e1c88fa336284c01#177">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/237543617bb645c09c5626a465ea9102?auth=1d9ec893f4db87f2a651ba9425af4413#106">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5dba910ce02146b683f0bc665c7e24dc?auth=6a35fa81e247524d8b9cf20e3332cb18#86">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590609">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bb6dc50eb5104160b6c31daf1c90c59d?auth=149859083ec55466e1c88fa336284c01#160">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/237543617bb645c09c5626a465ea9102?auth=1d9ec893f4db87f2a651ba9425af4413#78">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5dba910ce02146b683f0bc665c7e24dc?auth=6a35fa81e247524d8b9cf20e3332cb18#78">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590610">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bb6dc50eb5104160b6c31daf1c90c59d?auth=149859083ec55466e1c88fa336284c01#328">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/237543617bb645c09c5626a465ea9102?auth=1d9ec893f4db87f2a651ba9425af4413#203">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5dba910ce02146b683f0bc665c7e24dc?auth=6a35fa81e247524d8b9cf20e3332cb18#202">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590646">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/472490f1fa8a47debb10361baebdf320?auth=d0f272b61b544a2f65ce8d2c0e8f08c8#38">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590642">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1d46a573f6c94ccf8e4ed0a1de7873a1?auth=7e2fe995d40aaa47548a01362ade3e53#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590603">test_links_deep_links</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1d46a573f6c94ccf8e4ed0a1de7873a1?auth=7e2fe995d40aaa47548a01362ade3e53#203">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590629">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cb063a6655c14a1aa27f48ecc1aa834a?auth=b4ec66d739240ccede1281151953b53e#128">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/467ad3458bd94b6f881f277b8d9f1d6f?auth=127fe399ee8ed3aaed2ac0f01fa24ba7#101">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590630">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cb063a6655c14a1aa27f48ecc1aa834a?auth=b4ec66d739240ccede1281151953b53e#169">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/467ad3458bd94b6f881f277b8d9f1d6f?auth=127fe399ee8ed3aaed2ac0f01fa24ba7#117">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590632">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/76025d146f884680bf551db06179826f?auth=eb4fffcc77bedfc6fab948729889e8b9#229">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590601">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/76025d146f884680bf551db06179826f?auth=eb4fffcc77bedfc6fab948729889e8b9#76">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590625">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/76025d146f884680bf551db06179826f?auth=eb4fffcc77bedfc6fab948729889e8b9#105">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590620">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/76025d146f884680bf551db06179826f?auth=eb4fffcc77bedfc6fab948729889e8b9#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590636">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/76025d146f884680bf551db06179826f?auth=eb4fffcc77bedfc6fab948729889e8b9#124">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590611">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6791b41168d646beb3bb44bc016eaba1?auth=3114fb5a9cfab212b03201611f27da29#184">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0ef0623a0ba84a9597adc143017cf9a4?auth=8a7ead937f4e7e3e530e8fde34395f0f#168">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590608">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6791b41168d646beb3bb44bc016eaba1?auth=3114fb5a9cfab212b03201611f27da29#135">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0ef0623a0ba84a9597adc143017cf9a4?auth=8a7ead937f4e7e3e530e8fde34395f0f#110">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590619">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6791b41168d646beb3bb44bc016eaba1?auth=3114fb5a9cfab212b03201611f27da29#300">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/0ef0623a0ba84a9597adc143017cf9a4?auth=8a7ead937f4e7e3e530e8fde34395f0f#270">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590633">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/39acbc16ff4848ca8f172e2104d0c2bf?auth=b137c38f17c3007e7a4651c6ccef21be#227">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/429b55b3804741b184530b63f843e2f0?auth=fad00546334058d651220dc8196f6e83#164">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590624">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/39acbc16ff4848ca8f172e2104d0c2bf?auth=b137c38f17c3007e7a4651c6ccef21be#252">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/429b55b3804741b184530b63f843e2f0?auth=fad00546334058d651220dc8196f6e83#225">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590615">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/39acbc16ff4848ca8f172e2104d0c2bf?auth=b137c38f17c3007e7a4651c6ccef21be#300">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/429b55b3804741b184530b63f843e2f0?auth=fad00546334058d651220dc8196f6e83#285">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590631">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/39acbc16ff4848ca8f172e2104d0c2bf?auth=b137c38f17c3007e7a4651c6ccef21be#544">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/429b55b3804741b184530b63f843e2f0?auth=fad00546334058d651220dc8196f6e83#581">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590643">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/39acbc16ff4848ca8f172e2104d0c2bf?auth=b137c38f17c3007e7a4651c6ccef21be#579">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/429b55b3804741b184530b63f843e2f0?auth=fad00546334058d651220dc8196f6e83#607">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590614">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/39acbc16ff4848ca8f172e2104d0c2bf?auth=b137c38f17c3007e7a4651c6ccef21be#200">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/429b55b3804741b184530b63f843e2f0?auth=fad00546334058d651220dc8196f6e83#135">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590613">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/39acbc16ff4848ca8f172e2104d0c2bf?auth=b137c38f17c3007e7a4651c6ccef21be#158">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/429b55b3804741b184530b63f843e2f0?auth=fad00546334058d651220dc8196f6e83#106">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590618">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/39acbc16ff4848ca8f172e2104d0c2bf?auth=b137c38f17c3007e7a4651c6ccef21be#324">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/429b55b3804741b184530b63f843e2f0?auth=fad00546334058d651220dc8196f6e83#302">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590617">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/39acbc16ff4848ca8f172e2104d0c2bf?auth=b137c38f17c3007e7a4651c6ccef21be#176">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/429b55b3804741b184530b63f843e2f0?auth=fad00546334058d651220dc8196f6e83#123">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590616">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/39acbc16ff4848ca8f172e2104d0c2bf?auth=b137c38f17c3007e7a4651c6ccef21be#389">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/429b55b3804741b184530b63f843e2f0?auth=fad00546334058d651220dc8196f6e83#363">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590605">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8c4125bb760a4a688459078fbb0e7efb?auth=1f020de9ca1ae08e563ca2bc4dd7941a#149">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/247a03604ff744aa8b273acee595b48d?auth=b1ce6a7b2f8e62aba807e577cdaefdeb#119">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590600">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8c4125bb760a4a688459078fbb0e7efb?auth=1f020de9ca1ae08e563ca2bc4dd7941a#560">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/247a03604ff744aa8b273acee595b48d?auth=b1ce6a7b2f8e62aba807e577cdaefdeb#404">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590612">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8c4125bb760a4a688459078fbb0e7efb?auth=1f020de9ca1ae08e563ca2bc4dd7941a#419">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/247a03604ff744aa8b273acee595b48d?auth=b1ce6a7b2f8e62aba807e577cdaefdeb#259">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590602">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8c4125bb760a4a688459078fbb0e7efb?auth=1f020de9ca1ae08e563ca2bc4dd7941a#361">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/247a03604ff744aa8b273acee595b48d?auth=b1ce6a7b2f8e62aba807e577cdaefdeb#209">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590623">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8c4125bb760a4a688459078fbb0e7efb?auth=1f020de9ca1ae08e563ca2bc4dd7941a#468">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/247a03604ff744aa8b273acee595b48d?auth=b1ce6a7b2f8e62aba807e577cdaefdeb#290">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590637">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8c4125bb760a4a688459078fbb0e7efb?auth=1f020de9ca1ae08e563ca2bc4dd7941a#499">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/247a03604ff744aa8b273acee595b48d?auth=b1ce6a7b2f8e62aba807e577cdaefdeb#344">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590598">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8c4125bb760a4a688459078fbb0e7efb?auth=1f020de9ca1ae08e563ca2bc4dd7941a#211">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/247a03604ff744aa8b273acee595b48d?auth=b1ce6a7b2f8e62aba807e577cdaefdeb#164">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3590597">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8c4125bb760a4a688459078fbb0e7efb?auth=1f020de9ca1ae08e563ca2bc4dd7941a#95">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/247a03604ff744aa8b273acee595b48d?auth=b1ce6a7b2f8e62aba807e577cdaefdeb#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/quo/components/wallet/token_input/view.cljs
+++ b/src/quo/components/wallet/token_input/view.cljs
@@ -81,8 +81,9 @@
                                         (js->clj :keywordize-keys true)
                                         (on-selection-change))))]
     (fn [{:keys [token customization-color show-keyboard? crypto? currency value error? selection
-                 handle-on-swap]
-          :or   {show-keyboard? true}}]
+                 handle-on-swap allow-selection?]
+          :or   {show-keyboard?   true
+                 allow-selection? true}}]
       (let [theme        (quo.theme/use-theme)
             window-width (:width (rn/get-window))]
         [rn/pressable
@@ -107,7 +108,8 @@
                      :selection-color          customization-color
                      :show-soft-input-on-focus show-keyboard?
                      :on-selection-change      handle-selection-change
-                     :selection                (clj->js selection)}
+                     :selection                (clj->js selection)
+                     :editable                 allow-selection?}
               controlled-input?       (assoc :value value)
               (not controlled-input?) (assoc :default-value value-internal))]
            [token-name-text theme (if crypto? token currency)]]]

--- a/src/status_im/contexts/preview/quo/wallet/token_input.cljs
+++ b/src/status_im/contexts/preview/quo/wallet/token_input.cljs
@@ -23,6 +23,8 @@
     :options [{:key :usd}
               {:key :eur}]}
    {:key  :error?
+    :type :boolean}
+   {:key  :allow-selection?
     :type :boolean}])
 
 (defn view
@@ -33,7 +35,8 @@
                                  :networks            networks
                                  :title               title
                                  :customization-color :blue
-                                 :show-keyboard?      false})
+                                 :show-keyboard?      false
+                                 :allow-selection?    true})
         value     (reagent/atom "")
         set-value (fn [v]
                     (swap! value str v))

--- a/src/status_im/contexts/wallet/send/input_amount/view.cljs
+++ b/src/status_im/contexts/wallet/send/input_amount/view.cljs
@@ -349,23 +349,24 @@
        :on-press      on-navigate-back
        :switcher-type :select-account}]
      [quo/token-input
-      {:container-style style/input-container
-       :token           token-symbol
-       :currency        fiat-currency
-       :currency-symbol currency-symbol
-       :crypto-decimals (min token-decimals 6)
-       :error?          (controlled-input/input-error input-state)
-       :networks        (seq send-enabled-networks)
-       :title           (i18n/label
-                         :t/send-limit
-                         {:limit (if crypto-currency?
-                                   (utils/make-limit-label-crypto current-limit token-symbol)
-                                   (utils/make-limit-label-fiat current-limit currency-symbol))})
-       :conversion      conversion-rate
-       :show-keyboard?  false
-       :value           input-amount
-       :on-swap         swap-between-fiat-and-crypto
-       :on-token-press  show-select-asset-sheet}]
+      {:container-style  style/input-container
+       :token            token-symbol
+       :currency         fiat-currency
+       :currency-symbol  currency-symbol
+       :crypto-decimals  (min token-decimals 6)
+       :error?           (controlled-input/input-error input-state)
+       :networks         (seq send-enabled-networks)
+       :title            (i18n/label
+                          :t/send-limit
+                          {:limit (if crypto-currency?
+                                    (utils/make-limit-label-crypto current-limit token-symbol)
+                                    (utils/make-limit-label-fiat current-limit currency-symbol))})
+       :conversion       conversion-rate
+       :show-keyboard?   false
+       :value            input-amount
+       :on-swap          swap-between-fiat-and-crypto
+       :on-token-press   show-select-asset-sheet
+       :allow-selection? false}]
      [routes/view
       {:token                                     token-by-symbol
        :input-value                               input-amount

--- a/src/status_im/contexts/wallet/send/routes/view.cljs
+++ b/src/status_im/contexts/wallet/send/routes/view.cljs
@@ -170,22 +170,23 @@
            {:title       (i18n/label :t/send-from-network {:network network-name-str})
             :description (i18n/label :t/define-amount-sent-from-network {:network network-name-str})}]
           [quo/token-input
-           {:container-style style/input-container
-            :token           token-symbol
-            :currency        fiat-currency
-            :currency-symbol currency-symbol
-            :crypto-decimals (min token-decimals 6)
-            :error?          (controlled-input/input-error input-state)
-            :networks        [network-details]
-            :title           (i18n/label
-                              :t/send-limit
-                              {:limit (if crypto-currency?
-                                        (utils/make-limit-label-crypto current-limit token-symbol)
-                                        (utils/make-limit-label-fiat current-limit currency-symbol))})
-            :conversion      conversion-rate
-            :show-keyboard?  false
-            :value           (controlled-input/input-value input-state)
-            :on-swap         swap-between-fiat-and-crypto}]
+           {:container-style  style/input-container
+            :token            token-symbol
+            :currency         fiat-currency
+            :currency-symbol  currency-symbol
+            :crypto-decimals  (min token-decimals 6)
+            :error?           (controlled-input/input-error input-state)
+            :networks         [network-details]
+            :title            (i18n/label
+                               :t/send-limit
+                               {:limit (if crypto-currency?
+                                         (utils/make-limit-label-crypto current-limit token-symbol)
+                                         (utils/make-limit-label-fiat current-limit currency-symbol))})
+            :conversion       conversion-rate
+            :show-keyboard?   false
+            :value            (controlled-input/input-value input-state)
+            :on-swap          swap-between-fiat-and-crypto
+            :allow-selection? false}]
           [quo/disclaimer
            {:on-change           (fn [checked?]
                                    (set-is-amount-locked checked?))

